core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb_test_export.go

```diff
@@ -59,10 +60,33 @@ func (env *testVDBEnv) init(t testing.TB, cache *statedb.Cache) {
 }
 
 // startExternalResource sstarts external couchDB resources for testVDBEnv.
-func (env *testVDBEnv) startExternalResource() {
-	if env.couchAddress == "" {
-		env.couchAddress, env.cleanupCouchDB = couchdbtest.CouchDBSetup(nil)
+func (env *testVDBEnv) startExternalResource(t *testing.T) {
+	if env.couchAddress != "" {
+		return
+	}
+	// check if couchDB is being started externally.
+	externalCouchAddress, isSet := os.LookupEnv("COUCHDB_ADDR")
+	if isSet {
+		env.couchAddress, env.cleanupCouchDB = externalCouchAddress, func() {}
+		return
+	}
+
+	couchDB := &runner.CouchDB{}
+	require.NoError(t, couchDB.Start())
```

Comment:
This should be removed now as the next line is doing this

Reply:
Done.